src/commonComponent/Container/Container.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import './styles.css';
+import Counter from '../Counter/Counter'
+
+class Container extends Component {
+
+  constructor(props) {
+
+    super(props);
+
+    this.state = {
+
+      countValue: 0,
+      parityType: 'The number is even'
+    }
+
+  }
+
+  plusOne = () => {
```

Comment:
There are some naming conventions in React. If we are talking about functions which fires on user actions (click, change, etc.) - the best way to name it with "handle" word. 
Example: handleAdd or handlePlusOne in your case